network/p2p/scoring/score_option.go

```diff
@@ -151,28 +165,23 @@ func (s *ScoreOption) preparePeerScoreThresholds() {
 // preparePeerScoreParams prepares the peer score parameters for the pubsub system.
 // It is based on the default parameters defined in libp2p pubsub peer scoring.
 func (s *ScoreOption) preparePeerScoreParams() {
```

Comment:
```suggestion
func (s *ScoreOption) defaultPeerScoreParams() {
```

Reply:
Should we just return *pubsub.PeerScoreParams with default values instead of using this initialization pattern?